dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SystemSettingController.java

```diff
@@ -235,6 +232,8 @@ private boolean settingExistsAndIsNotConfidential(String key) {
   private Serializable getSystemSettingOrTranslation(String key, String locale, User currentUser) {
     if (settingExistsAndIsNotConfidential(key)) {
       Optional<SettingKey> settingKey = SettingKey.getByName(key);
+      if (settingKey.isEmpty()) return StringUtils.EMPTY;
```

Comment:
returning an empty string is the pattern applied in this method when something not found so keeping it consistent here